intermediate_source/scaled_dot_product_attention_tutorial.py

```diff
@@ -0,0 +1,333 @@
+"""
+Implement High-Performance Transformers with SCALED DOT PRODUCT ATTENTION
```

Comment:
super minor, but if we follow naming from other PyTorch tutorials / recipes, I think "Implementing" would work better here